accounts.go

```diff
@@ -42,7 +42,11 @@ func (srv *accountsService) CreateAccount(ctx context.Context, in *accountspb.Cr
 		return nil, status.Errorf(codes.Internal, "could not create account")
 	}
 
-	srv.repo.Create(ctx, &models.AccountPayload{Email: &in.Email, Name: &in.Name, Hash: &hashed})
+	err = srv.repo.Create(ctx, &models.AccountPayload{Email: &in.Email, Name: &in.Name, Hash: &hashed})
+	if err != nil {
+		srv.logger.Errorw("failed to create account", "error", err.Error())
```

Comment:
I discovered the other day that you can do the following 😊
```suggestion
		srv.logger.Errorw("failed to create account", zap.Error(err))
```